cpp/CMakeLists.txt

```diff
@@ -36,6 +36,8 @@ if(CMAKE_CUDA_COMPILER_ID STREQUAL "NVIDIA" AND CMAKE_CUDA_COMPILER_VERSION VERS
   )
 endif()
 
+rapids_cmake_write_version_file(include/cudf/version_config.hpp)
```

Comment:
We need an install rule for this file

Reply:
Oh whoops good call. I somehow though that was automatically handled by the rapids-cmake function.